pkg/background/request_process.go

```diff
@@ -34,24 +38,38 @@ func (c *Controller) MarkUR(ur *urkyverno.UpdateRequest) (*urkyverno.UpdateReque
 		}
 		return ur, true, nil
 	}
-
 	handler = config.KyvernoPodName
 	ur.Status.Handler = handler
```

Comment:
If we switch to use Patch, do we need to set the handler here?

Reply:
missed to remove